web/themes/contrib/civictheme/templates/form/fieldset.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * CivicTheme theme implementation to display a fieldset.
+ */
+#}
+{% include '@atoms/fieldset/fieldset.twig' with {
+  legend: legend.title
```

Comment:
@govindmaloo 
please advise why this change is needed

Reply:
This is required because the default legend array has a title and attributes elements and an invalid render array attributes error was showing on the page.